azure-pipelines/templates/stages/build.yml

```diff
@@ -79,4 +79,23 @@ stages:
       displayName: Pagkage executables
 
     - publish: $(Build.ArtifactStagingDirectory)/package-$(target)
-      artifact: package-$(target)
\ No newline at end of file
+      artifact: package-$(target)
+
+    - job: PublishScripts
+      dependsOn: Build
+      condition: succeeded()
+      steps:
+        - checkout: self
+        - script: |
+            echo Copying installation scripts to artifacts directory...
+            cp ./scripts/install.cmd $(Build.ArtifactStagingDirectory)/scripts/
+            cp ./scripts/install.sh $(Build.ArtifactStagingDirectory)/scripts/
+            echo Copying uninstallation scripts to artifacts directory...
+            cp ./scripts/uninstall.cmd $(Build.ArtifactStagingDirectory)/scripts/
+            cp ./scripts/uninstall.sh $(Build.ArtifactStagingDirectory)/scripts/
+            echo Copying test scripts to artifacts directory...
+            cp ./scripts/test.cmd $(Build.ArtifactStagingDirectory)/scripts/
+            cp ./scripts/test.sh $(Build.ArtifactStagingDirectory)/scripts/
```

Comment:
We should put windows and linux scripts in their own artifacts I think `scripts-win` and `scripts-linux` and also publish the linux scripts as `scripts-macos` since we also use the same scripts there

Reply:
I did it in the last changes so I have separated the script into two separate tasks (jobs) in the `build.yml` :

- PublishScripts -handle scripts for Linux and macOS, also in steps of this job the scripts are copied to the corresponding directory for artefacts and published under the names `scripts-linux` and `scripts-macos` as you mentioned
- PublishWindowsScripts - handle scripts for Windows systems, similar like above but published under the name `scripts-windows`